src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts

```diff
@@ -31,8 +31,45 @@ const getTopMostReportIDFromRHP = (state: State): string => {
     return '';
 };
 
+// Function that checks if the given route has a policyID equal to the id provided in the function params
```

Comment:
When commenting functions, we don't focus on the fact that they're functions. We either write "Do something such and such" or "Does something and something".